### DISCUSSION TITLE
Add the extension to firefox addon store

### DISCUSSION THREAD

**Discussion by Midnight0507** [HASHCUT69/Blend-n-Run#28] [Ideas]
this extension is amazing for pasting code in tesselator ,  you guys should try to implement this extension to firefox users in kmit would heavily appreciate it

**Comment by sathvik-lab** [comment]
Sure @CyrilTheLearner!! We are planning to build it for firefox users, keep an eye out for new releases and make sure you spread the word.

**Comment by sathvik-lab** [comment]
Try to contribute to the newest release when it comes out.
